spec/models/target_spec.rb

```diff
@@ -27,6 +27,21 @@
     it { is_expected.to validate_presence_of(:radius) }
     it { is_expected.to validate_presence_of(:latitude) }
     it { is_expected.to validate_presence_of(:longitude) }
+
+    describe 'amount of targets limit reached' do
+      let(:user) { create(:user) }
+      let!(:targets) { create_list(:target, Target::USER_MAX_TARGET_LIMIT, user_id: user.id) }
+      let(:target) { build(:target, user_id: user.id) }
+
+      it 'it does not create a new target' do
+        expect(target).to_not be_valid
```

Comment:
Actually, the user should be able to create 10 targets. 11 should be an error.

Reply:
I think is the 11 one, because before that it is created and asociated to the user the maximum quantity.

`let!(:targets) { create_list(:target, Target::USER_MAX_TARGET_LIMIT, user_id: user.id) }`